kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/OffsetFinder.java

```diff
@@ -64,7 +64,7 @@ public void findMessages(final long timestamp, AsyncCallbacks.FindEntryCallback
                     msg = MessageImpl.deserialize(entry.getDataBuffer());
                     return msg.getPublishTime() <= timestamp;
                 } catch (Exception e) {
-                    log.error("[{}][{}] Error deserializing message for message position find",  e);
+                    log.error("Error deserialize message for message position find",  e);
```

Comment:
also here, logged the information of managedLedger.getName() and timestamp is useful